_templates/exchanges.html

```diff
@@ -65,6 +65,7 @@ <h3 id="brazil"><img src="/img/flags/BR.png" alt="Brazilian flag">Brazil</h3>
       <p>
         <a href="https://foxbit.com.br/">Foxbit</a><br>
         <a href="https://www.mercadobitcoin.com.br/">Mercado Bitcoin</a>
```

Comment:
A line break (`<br>`) is missing after Mercado Bitcoin. Without it, the listings will break - could you please fix?